drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -126,16 +129,18 @@ public void setCopyFontType(String fontPath) {
 
         hasPaintChanged = true;
         requestLayout();
+        invalidate();
     }
 
-    public void setDropCapTextSize(@DimenRes float textSize) {
+    public void setDropCapTextSize(float textSize) {
```

Comment:
Thanks for asking the question @eduardb as @mr-archano says it is in `px`, I'll update the parameter to be more explicit. I'll add the overload too, as you suggested @mr-archano 👍